src/go/rpk/pkg/api/admin/admin.go

```diff
@@ -243,13 +243,14 @@ func (a *AdminAPI) GetLeaderID() (*int, error) {
 	return &pa.LeaderID, nil
 }
 
-// sendAny sends a single request to one of the client's urls and unmarshals
-// the body into into, which is expected to be a pointer to a struct.
+// sendAny sends a single request to one of the client's urls, return the host
+// that was queried, and unmarshals the body into into, which is expected to be
+// a pointer to a struct.
 //
 // On errors, this function will keep trying all the nodes we know about until
 // one of them succeeds, or we run out of nodes.  In the latter case, we will return
 // the error from the last node we tried.
-func (a *AdminAPI) sendAny(method, path string, body, into interface{}) error {
+func (a *AdminAPI) sendAny(method, path string, body, into interface{}) (string, error) {
```

Comment:
What do you think about keeping sendAny, adding sendAnyWithHost, and having sendAny call sendAnyWithHost and return only the error? This would avoid changes in everything that uses sendAny (minus the one spot we want the host)